log/log.go

```diff
@@ -30,18 +32,80 @@ var levelPrefix = [...]string{
 	LevelFatal:    "[FATAL] ",
 }
 
+var levelPriority = [...]syslog.Priority{
+	LevelDebug:    syslog.LOG_DEBUG,
+	LevelInfo:     syslog.LOG_INFO,
+	LevelWarning:  syslog.LOG_WARNING,
+	LevelError:    syslog.LOG_ERR,
+	LevelCritical: syslog.LOG_CRIT,
+	LevelFatal:    syslog.LOG_EMERG,
+}
+
 // Level stores the current logging level.
-var Level = LevelDebug
+var Level = LevelInfo
+
+var (
+	// SysLoggers maps each logging level to a *log.Logger that will be used for it.
+	SysLoggers = make([]*golog.Logger, LevelFatal+1)
```

Comment:
Point. Correcting now.

A limitation of the log/syslog package is that each syslog writer can only have one priority. I'd prefer to have one syslogger for each priority, so that priority information is available to the syslog server.